### PR TITLE
Feat/198 - main page

### PR DESCRIPTION
**Pull request by qdozaq**

### CODE REVIEW

File: src/app/page/Recipient/Dashboard.jsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export default function () {
+  return <p>hey</p>;
```
Comment (utunga) [inline src/app/page/Recipient/Dashboard.jsx]: Uh? Maybe not supposed to add this file?

File: src/app/page/Recipient/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import Dashboard from "./Dashboard";
+
```
Comment (utunga) [inline src/app/page/Recipient/index.js:2]: Should we be importing this?

File: src/app/page/Recipient/Dashboard.jsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  }[path] || 0);
+
+export default function () {
+  const classes = useStyles();
+  const location = useLocation();
+  const auth = useSelector((state) => state.firebase.auth);
+  const [missions, setMissions] = useState({ submitted: [], completed: [] });
+
+  useEffect(() => {
+    User.getAllRequstedMissions(auth.uid).then((missions) => {
```
Comment (mat10tng) [inline src/app/page/Recipient/Dashboard.jsx:44]: spelling ? getAl**lRequsted**Missions

File: src/app/page/Recipient/Dashboard.jsx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+      const submitted = missions.filter(
+        ({ status }) => !(status === MissionStatus.delivered || status === MissionStatus.succeeded)
```
Comment (mat10tng) [inline src/app/page/Recipient/Dashboard.jsx:46]: includes instead of equalities?

File: src/app/page/Recipient/Dashboard.jsx [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+            <RequestsList missions={missions.completed} />
+          </Route>
+        </Switch>
+      </Box>
+      <Button
+        className={classes.requestButton}
+        size="large"
+        variant="contained"
+        color="primary"
+        startIcon={<Add />}
```
Comment (mat10tng) [inline src/app/page/Recipient/Dashboard.jsx:103]: looks good, I see a few others who add icon in the actual child here instead.

**Pull request merged**